### PR TITLE
Benchmark labels added to CPU Logistic regression

### PR DESCRIPTION
**Pull request by michael-smirnov**
This PR contains additional marks that are used for kernel profiling on the benchmarks level.

### CODE REVIEW

File: cpp/daal/src/algorithms/objective_function/logistic_loss/logistic_loss_dense_default_batch_impl.i [hunk 1]
```diff
@@ -395,6 +397,8 @@ services::Status LogLossKernel<algorithmFPType, method, cpu>::doCompute(const al
 
         if (valueNT)
         {
+            DAAL_ITTNOTIFY_SCOPED_TASK(computeLogProbabilities);
```
Comment (PetrovKP) [inline cpp/daal/src/algorithms/objective_function/logistic_loss/logistic_loss_dense_default_batch_impl.i:400]: Why `computeLogProbabilities`?

Comment (michael-smirnov) [reply cpp/daal/src/algorithms/objective_function/logistic_loss/logistic_loss_dense_default_batch_impl.i]: changed to computeValueResult

**Review by PetrovKP — APPROVED** [review]

**Review by masdevas — APPROVED** [review]

**Pull request merged**